manage.py

```diff
@@ -1,9 +1,45 @@
-#!/usr/bin/env python
 """Django's command-line utility for administrative tasks."""
-import os
 import sys
+import os
```

Comment:
Keep the entries sorted. Also, making formatting changes in the same PR as functional changes is considered a very bad tone because this makes it _much harder_ to compare the versions during review.
```suggestion
import os
import sys
```

Reply:
Marking this as resolved.